code/modules/food_and_drinks/kitchen_machinery/kitchen_machine.dm

```diff
@@ -156,8 +156,60 @@
 		//G.reagents.trans_to(src,G.amount_per_transfer_from_this)
 	else if(istype(O,/obj/item/weapon/grab))
 		var/obj/item/weapon/grab/G = O
-		to_chat(user, "<span class='alert'>This is ridiculous. You can not fit \the [G.affecting] in this [src].</span>")
-		return 1
+		if(!istype(src, /obj/machinery/kitchen_machine/oven) && !istype(src, /obj/machinery/kitchen_machine/grill))
```

Comment:
This is the coincide with the message given that you can't do anything with it vs which ones you CAN without giving error of any of the other types (such as attempting to put someone in the microwave which will never work in any regard this moment). It's more user-friendly side of 'stop trying'.